__tests__/app.test.js

```diff
@@ -79,3 +78,57 @@ describe("GET api/reviews/review_id", () => {
       });
   });
 });
+
+describe("PATCH api/reviews/:review_id", () => {
+  test("Status 200 - Responds with review and updated vote count", () => {
+    const bodyToSend = { inc_votes: 15 };
+    return request(app)
+      .patch("/api/reviews/3")
+      .send(bodyToSend)
+      .expect(201)
+      .then(({ body : { review } }) => {
+        expect.objectContaining({
+            review_id: expect.any(Number),
+            title: expect.any(String),
+            review_body: expect.any(String),
+            designer: expect.any(String),
+            review_img_url: expect.any(String),
+            votes: expect.any(Number),
+            category: expect.any(String),
+            owner: expect.any(String),
+            created_at: expect.any(String),
+          });
+        expect(review.votes).toBe(20)
+      });
```

Comment:
Here you are slightly mixing up asserting structure rather than value and sort of doing both. I would probably assert the actual review, with values rather than data types. Then you don't need the extra expect statement